storage/timecache/timeCacheCore.go

```diff
@@ -58,7 +58,7 @@ func (tcc *timeCacheCore) upsert(key string, value interface{}, duration time.Du
 }
 
 // put will add the key, value and provided duration, overriding values if the data already existed
-// It returns if the value existed before this call. It also operates on the locker so the call is concurrent safe
+// It returns true if the value existed before this call. It also operates on the locker so the call is concurrent safe
 func (tcc *timeCacheCore) put(key string, value interface{}, duration time.Duration) (bool, error) {
```

Comment:
Why have this func return true if the value existed, since in `func (tc *timeCacher) Put` you always return false and ignore if this one returns true?

Reply:
removed & optimized

Reply:
`timeCacheCore` should normally not care about the existence, nor the logic of its caller (`timeCacher`). Therefore, it's alright to provide the information to the caller - even if that piece of information isn't used.